packages/dd-trace/src/priority_sampler.js

```diff
@@ -150,7 +202,15 @@ class PrioritySampler {
       : this._getPriorityByAgent(context)
   }
 
-  _getPriorityFromTags (tags) {
+  /**
+   *
+   * @param tags {Map<string, Object>}
```

Comment:
I don't know exactly what JSDoc refers to when it being an Object but at least in TS it's not ideal to use. Should we use something like `Record<string | symbol, unknown>`?